Autoconf fails with the libltdl stuff, so just ignore and use it anyways

--- old/lib/libltdl/Makefile.am
+++ new/lib/libltdl/Makefile.am
@@ -50,19 +50,7 @@
 LTDL_VERSION_INFO	= -version-info 9:0:2
 
 noinst_LTLIBRARIES	+= $(LT_DLLOADERS)
-
-if INSTALL_LTDL
-ltdlincludedir		= $(includedir)/libltdl
-ltdlinclude_HEADERS	= libltdl/lt_system.h \
-			  libltdl/lt_error.h \
-			  libltdl/lt_dlloader.h
-include_HEADERS	       += ltdl.h
-lib_LTLIBRARIES	       += libltdl.la
-endif
-
-if CONVENIENCE_LTDL
 noinst_LTLIBRARIES     += libltdlc.la
-endif
 
 libltdl_la_SOURCES = libltdl/lt__alloc.h \
 			  libltdl/lt__dirent.h \
